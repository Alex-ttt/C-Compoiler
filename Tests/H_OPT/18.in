void main(){
	int a;
	if(a > 5){
		4 + 6;
		a++;
		return;
		a * 5;
	} else 
	{
		4 - 6;
		a--;
		return;
		a / 5;
	}
}