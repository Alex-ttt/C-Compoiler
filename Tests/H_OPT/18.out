 Symbols Table:
   main  void function main(
  )
    {
      Statements:
      if

          a
        >
          5

      then

        {
          Statements:
            4
          +
            6
          a
          ++
          return
            a
          *
            5
        }
      else

        {
          Statements:
            4
          -
            6
          a
          --
          return
            a
          /
            5
        }
      Symbols Table:
              a int
    }
